cmd/root.go

```diff
@@ -47,42 +47,43 @@ var (
 
 type Submission struct {
 	URL string `json:"url"`
+	ShortCode string `json:"short_code"`
 }
 
 type Response struct {
 	URL   string `json:"url"`
 	ID    string `json:"id"`
+	ShortCode string `json:"short_code"`
 	Error string `json:"error"`
 }
 
 const track = "?WT.mc_id"
 
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
-	Use:   "cda URL",
+	Use:   "cda URL ShortCode",
```

Comment:
to fit the style of the example usage, this should be "cda URL SHORTCODE"

Reply:
Cool, can do :)